sales.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I actually didn't know about the `lang` attribute, but MDN recommends to always include it to help browsers and search engines. Good addition!